src/services/ExplorerService/fetcher.ts

```diff
@@ -156,6 +160,7 @@ export const apiFetcher = {
       | Response.Wrapper<Address, SearchResultType.Address>
       | Response.Wrapper<Address, SearchResultType.LockHash>
       | Response.Wrapper<unknown, SearchResultType.UDT>
+      | SearchTypeIdResult
```

Comment:
Why does the backend provide such a different wrapping structure?

Reply:
Is it convenient to return a 
```
{
  "id": ...,
  "type": ...,
  "attribute": ...
}
```
like structure? @rabbitz